subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/serialization/Workarounds.kt

```diff
@@ -18,27 +18,19 @@ package org.gradle.instantexecution.serialization
 
 import org.gradle.api.internal.initialization.ClassLoaderScope
 import java.lang.reflect.Field
-import java.util.concurrent.ForkJoinPool
 
 
 internal
 object Workarounds {
 
     private
-    val ignoredBeanFields = listOf(
-        // Ignore a lambda field for now
-        "mFolderFilter" to "com.android.ide.common.resources.DataSet"
-    )
+    val ignoredBeanFields: List<Pair<String, String>> = emptyList()
 
     fun isIgnoredBeanField(field: Field) =
         ignoredBeanFields.contains(field.name to field.declaringClass.name)
```

Comment:
shouldn't this just be `= false`? which means we can remove the method.

Reply:
Yep, I'll remove `Workarounds` altogether.